package.json

```diff
@@ -25,6 +25,7 @@
   "scripts": {
     "compile": "tsc",
     "generateDocs": "npm run compile && node dist/src/docGeneratorMain.js",
+    "start": "npm run compile && node dist/src/index.js",
```

Comment:
This is useless since the tool requires thing from the environment (`/src` directory etc.)

Reply:
Removed it